packages/cysync-core/src/dialogs/EditAccountDialog/Dialogs/AccountDetails.tsx

```diff
@@ -0,0 +1,240 @@
+import {
+  ArrowDown,
+  ArrowUp,
+  Button,
+  CloseButton,
+  Container,
+  DialogBox,
+  DialogBoxBody,
+  DialogBoxFooter,
+  DialogBoxHeader,
+  Divider,
+  EditAccountIcon,
+  Flex,
+  Input,
+  JsonView,
+  LangDisplay,
+  MessageBox,
+  ScrollContainer,
+  ScrollableContainer,
+  Typography,
+  parseLangTemplate,
+} from '@cypherock/cysync-ui';
+import React, { useEffect, useMemo, useState } from 'react';
+
+import { openDeleteAccountDialog } from '~/actions';
+import { selectLanguage, useAppDispatch, useAppSelector } from '~/store';
+
+import { useEditAccountDialog } from '../context';
+
+export const AccountDetails: React.FC = () => {
+  const lang = useAppSelector(selectLanguage);
+
+  const {
+    onClose,
+    onApply,
+    selectedAccount,
+    selectedWallet,
+    accountName,
+    handleAccountNameChange,
+    // unitDropdownList,
+    // selectedUnit,
+    // setSelectedUnit,
+  } = useEditAccountDialog();
+
+  const { accountEdit, header: headerText } = lang.strings.dialogs.editAccount;
+  const [showAdvance, setShowAdvance] = useState(false);
+  const dispatch = useAppDispatch();
+
+  // const handleUnitChangeProxy: typeof setSelectedUnit = useCallback(
+  //   (...args) => {
+  //     logger.info('Dropdown Change: Unit Change', {
+  //       source: `EditAccount/${AccountDetails.name}`,
+  //       unit: args[0],
+  //     });
+  //     return setSelectedUnit(...args);
+  //   },
+  //   [setSelectedUnit],
+  // );
+
+  const accountData = useMemo(
+    () => ({
+      derivationPath: selectedAccount?.derivationPath,
+      derivationScheme: selectedAccount?.derivationScheme,
+      xpubOrAddress: selectedAccount?.xpubOrAddress,
+    }),
+    [selectedAccount],
+  );
+
+  const accountNameInputRef = React.useRef<HTMLInputElement>(null);
+  useEffect(() => {
+    if (accountNameInputRef.current) {
+      accountNameInputRef.current.focus();
+    }
+  }, [accountNameInputRef]);
```

Comment:
It's better to move this code to `Input` component and add a prop like `autoFocus`.